source/operator.rst

```diff
@@ -208,72 +258,21 @@ The replsets section controls the MongoDB Replica Set.
        - ``Directory``
        - The `Kubernetes hostPath volume type <https://kubernetes.io/docs/concepts/storage/volumes/#hostpath>`_
 
-     * - volumeSpec.persistentVolumeClaim.storageClassName
+     * - volumeSpec.persistentVolumeClaim. storageClassName
        - string
```

Comment:
what is the space for?

Reply:
this space makes a line break. Because of it 1st column is not the widest one, and some space is saved for description